tests/torchtune/models/mistral/scripts/compare_mistral_classifier.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Thanks so much for adding this! You really went above and beyond here. Actually one more change request here (this one should be much easier, I promise). Can you add back the previous unit test you deleted in the latest commit? I think my previous wall of text could've been a bit clearer, but the main point I was getting at was that the values from this script can then be used to set the expected values in the unit test.

The idea being that then we will actually run the unit test in our CI, whereas this script is more of a nice-to-have so we can understand the initial testing that went into setting the unit test values. In case it helps, I ran your script locally myself: [here's the output](https://gist.github.com/ebsmothers/ba220949626911788282f4b338f354a6) I got so that you don't have to run it again. And [here](https://gist.github.com/ebsmothers/031a88763285454d342cf197d4dad900) is roughly what I think the unit test will look like now (hopefully not too many bugs in there 😅). Again, sorry for the back and forth on this!

Reply:
I get it now :)